.gitignore

```diff
@@ -24,3 +24,4 @@
 /doc/
 /failures
 .DS_Store
+.byebug_history
```

Comment:
Could this be added into a separate commit to ensure the commit message ties up with the code change?

Reply:
Created a separate commit for this.